app/models/manageiq/providers/vmware/infra_manager/refresher.rb

```diff
@@ -15,7 +15,7 @@ def self.init_console(use_vim_broker = false)
         def provider.use_vim_broker?; @__use_vim_broker; end
         klass = use_vim_broker ? MiqVimBroker : MiqVimInventory
         klass.cacheScope = :cache_scope_ems_refresh
-        klass.setSelector(EmsRefresh::VcUpdates::VIM_SELECTOR_SPEC)
+        klass.setSelector(ManageIQ::Providers::Vmware::InfraManager::SelectorSpec::VIM_SELECTOR_SPEC)
```

Comment:
you _could_ use parent to avoid the FQN 
```
klass.setSelector(parent::SelectorSpec::VIM_SELECTOR_SPEC)
```

but it looks like we seldom use this...